src/cli/cmds/new.ts

```diff
@@ -95,16 +96,17 @@ class Command implements yargs.CommandModule {
   }
 
   public async handler(args: any) {
-    console.log(`Invalid project type ${args.projectType}. Supported types:`);
-    for (const pjid of inventory.discover().map(x => x.pjid)) {
-      console.log(`  ${pjid}`);
+    if (args.from && args.from !== '') {
```

Comment:
In javascript, an empty string is falsy:

```suggestion
    if (args.from) {
```